src/mon/HealthMonitor.cc

```diff
@@ -67,8 +67,8 @@ static ostream& _prefix(std::ostream *_dout, const Monitor *mon,
 		<< "(" << mon->get_state_name() << ").health ";
 }
 
-HealthMonitor::HealthMonitor(Monitor *m, Paxos *p, const string& service_name)
-  : PaxosService(m, p, service_name) {
+HealthMonitor::HealthMonitor(Monitor *m, Paxos *p)
+  : PaxosService(m, p, get_paxos_name<PAXOS_HEALTH>()) {
```

Comment:
i am not in favor of this change. as it trades readability for flexibility which is not necessary here for obvious reasons.

Reply:
I'll abandon this commit since I decided to remove get_paxos_name function.